speed_limiter is not compiling on macOS Catalina

I got the following compiling error on macOS Catalina:
`
diff_drive_controller/src/speed_limiter.cpp:63:9: error: no member named 'isnan' in namespace 'std'; did you mean simply 'isnan'?
    if (std::isnan(min_jerk_)) {
        ^~~~~~~~~~
        isnan
`
The reason is a missing `cmath` in the set of inclusion.

`math.h` is included is speed_limiter.hh

But according to cppreference the header is cmath.

--- a/diff_drive_controller/include/diff_drive_controller/speed_limiter.hpp
+++ b/diff_drive_controller/include/diff_drive_controller/speed_limiter.hpp
@@ -19,7 +19,7 @@
 #ifndef DIFF_DRIVE_CONTROLLER__SPEED_LIMITER_HPP_
 #define DIFF_DRIVE_CONTROLLER__SPEED_LIMITER_HPP_
 
-#include <math.h>
+#include <cmath>
 
 namespace diff_drive_controller
 {

